.github/workflows/pr-title-linting.yml

```
@@ -0,0 +1,33 @@
+name: Lint PR Title
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: PR Title Checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5.1.0
+        with:
+          types: |
+            chore
+            docs
+            feat
+            fix
+            refactor
+            style
+            perf
+            ci
+            test
+            revert
```

Just a couple more suggestions:
```suggestion
            revert
            integration
            misc
```

---

Oh I see you made it a scope.

---

yes, I think that makes sense in the current structure :)